UI/OptionsWnd.cpp

```diff
@@ -375,6 +376,68 @@ namespace {
             LockColWidths();
         }
     };
+
+
+    /**Create UI controls for a logger level option.*/
+    void LoggerLevelOption(
```

Comment:
if you have such a long list of parameters, you might as well start them on the line with the function name, and break them over two lines, but not going so far right on either line...